### DISCUSSION TITLE
mi cuenta principal se malogro

### DISCUSSION THREAD

**Discussion by ramirezherreranoeelvis** [community/community#103067] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Es GatomontesRoseIII
cuando intento visitarla me sale el 404
esta es otra cuenta nueva porque no hay solución creo, si la puedo usar pero no la puedo visitar
![image](https://github.com/community/community/assets/158113337/9f5081e5-5f03-4218-833e-e227fcd8cd82)
![image](https://github.com/community/community/assets/158113337/d2f7fd48-5586-4303-bb6d-8fb0239dcbc3)

**Comment by ramirezherreranoeelvis** [comment]
j

**Comment by queenofcorgis** [comment]
Hi @nramirezhelvis , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
